GeneratedRegexPatterns/UnclassifiedPotentialSecurityKeys.json

```diff
@@ -45,34 +45,34 @@
     "Id": "SEC000/003",
     "Name": "Unclassified32CharacterString",
     "Signatures": null,
-    "DetectionMetadata": "ObsoleteFormat, HighEntropy, Unclassified"
```

Comment:
>ObsoleteFormat [](http://example.com/codeflow?start=26&length=14)

This pattern *used* to cover AAD 32 character passwords, which are obsolete in format. We no longer actually cover this legacy pattern, which is five generations out as a format, extremely noisy, etc.